src/main/java/org/gridsuite/gateway/NettyMetricsConfiguration.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.gateway;
+
+import io.micrometer.core.instrument.config.MeterFilter;
+import org.springframework.boot.web.embedded.netty.NettyServerCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import reactor.netty.http.server.HttpServer;
+
+import java.util.function.Function;
+
+/**
+ * @author Seddik Yengui <seddik.yengui_externe at rte-france.com>
+ */
+
+// the metrics expose by reactor-netty are not enabled by default in spring boot.
+// To enable them we add the following bean to customise the Netty HttpServer.
```

Comment:
// Enable Netty metrics that are not enabled by default in Spring Boot.

Reply:
done